samples/unity-of-bugs/Assets/Resources/Sentry/SentryOptions.asset

```diff
@@ -41,17 +41,17 @@ MonoBehaviour:
   <SampleRate>k__BackingField: 1
   <ShutdownTimeout>k__BackingField: 2000
   <MaxQueueItems>k__BackingField: 30
+  <AnrDetectionEnabled>k__BackingField: 1
+  <AnrTimeout>k__BackingField: 5000
   <IosNativeSupportEnabled>k__BackingField: 1
   <AndroidNativeSupportEnabled>k__BackingField: 1
   <WindowsNativeSupportEnabled>k__BackingField: 1
   <MacosNativeSupportEnabled>k__BackingField: 1
   <LinuxNativeSupportEnabled>k__BackingField: 1
   <Il2CppLineNumberSupportEnabled>k__BackingField: 1
-  <AnrDetectionEnabled>k__BackingField: 1
-  <AnrTimeout>k__BackingField: 5000
-  <OptionsConfiguration>k__BackingField: {fileID: 11400000, guid: 407b4d5f2fef2c845bf2a3dcdf6b00fb,
+  <RuntimeOptionsConfiguration>k__BackingField: {fileID: 11400000, guid: f9a47e8442a604414a84d5ad6da6aa43,
     type: 2}
-  <BuildtimeOptionsConfiguration>k__BackingField: {fileID: 11400000, guid: 6bcc81a646c08ce439ab03805d477458,
+  <BuildTimeOptionsConfiguration>k__BackingField: {fileID: 11400000, guid: 1c61999bd2a61408393cfe218d42287e,
```

Comment:
is this a breaking change? If a user upgrades the SDK do they lose their option?

Reply:
Yes. Unfortunately. I aim to include this as the `1.0.0` release to make the breaking change more palatable.